doc: Added explanation of chunk_overlap to knowledge API

yagiyuki opened the pull request:
# Summary

The following four Knowledge APIs allow you to specify "chunk_overlap" as a parameter when creating a document.
However, this is not described in the API specifications, so it needs to be added.

* Create a Document from Text
* Create a Document from a File
* Update a Document with Text
* Update a Document with a File

Fixes https://github.com/langgenius/dify/issues/12180

> [!Tip]
> Close issue syntax: `Fixes #<issue number>` or `Resolves #<issue number>`, see [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more details.


# Screenshots

| Before | After |
|--------|-------|
| ...    | ...   |

# Checklist

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [x] I understand that this PR may be closed in case there was no previous discussion or issues. (This doesn't apply to typos!)
- [x] I've added a test for each change that was introduced, and I tried as much as possible to make a single atomic change.
- [x] I've updated the documentation accordingly.
- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

jasonfish568 commented:
Commit message not up to the community standard. I would suggest you do some readings on commit message format and also properlly include description in the PR.

yagiyuki commented:
> Commit message not up to the community standard. I would suggest you do some readings on commit message format and also properlly include description in the PR.

@jasonfish568 

I have updated the PR Summary to include a more detailed description of the changes. Regarding the commit message format, I will make sure to follow the community standards for future commits.

crazywoola commented:
Hello there, please update the docs in JA as well.

yagiyuki commented:
@crazywoola 
> Hello there, please update the docs in JA as well.

Thank you for your feedback. Currently, there is no documentation in Japanese for the Knowledge API.
https://github.com/langgenius/dify/tree/5b01eb9437b24a260bf4ec3084256d9995bcf93a/web/app/(commonLayout)/datasets/template

Therefore, I don't think there is any need to take action.

crazywoola committed:
Resolve: conflict

crazywoola commented:
> @crazywoola
> 
> > Hello there, please update the docs in JA as well.
> 
> Thank you for your feedback. Currently, there is no documentation in Japanese for the Knowledge API. https://github.com/langgenius/dify/tree/5b01eb9437b24a260bf4ec3084256d9995bcf93a/web/app/(commonLayout)/datasets/template
> 
> Therefore, I don't think there is any need to take action.
> 
> If you think adding it would be helpful, feel free to create a new issue—or let me know, and I can create one.

Thanks :)

Actually I figure it would be helpful to add JA document to this section, because we have added several templates in API develop section right now.

https://github.com/langgenius/dify/blob/6b49889041f5d25cd2c5bc2e0f7915abb7027027/web/app/components/develop/template/template.ja.mdx

We appreciate that you could add this if you have some spare time.

yagiyuki commented:
@crazywoola 

マージありがとう。

> Actually I figure it would be helpful to add JA document to this section, because we have added several templates in API develop section right now.
> 
> https://github.com/langgenius/dify/blob/6b49889041f5d25cd2c5bc2e0f7915abb7027027/web/app/components/develop/template/template.ja.mdx
> 
> We appreciate that you could add this if you have some spare time.

I added a Japanese-language issue to the Knowledge API.
https://github.com/langgenius/dify/issues/12359
